### PR TITLE
[RN] Release dance

### PR DESCRIPTION
**Pull request by turtledreams**

### CODE REVIEW

File: react_native_bridged/current.md [hunk 1]
```diff
@@ -64,59 +67,42 @@ if(!await Countly.isInitialized()) {<br>  // create Countly config object
...
+  const countlyConfig = new CountlyConfig("https://try.count.ly", "YOUR_APP_KEY");
+  // ...
   countlyConfig.setLoggingEnabled(true); // Enable countly internal debugging logs
   await Countly.initWithConfig(countlyConfig); // Initialize the countly SDK with config.
   </code></pre>
 <p>
   For more information on where to find the SDK logs you can check the documentation
   <a href="https://support.count.ly/hc/en-us/articles/900000908046-Getting-started-with-SDKs#finding-sdk-logs" target="blank">here</a>.
 </p>
-<h2 id="h_01HAVQNJQQE2NNR8AVVZD1EHNT">Device ID</h2>
```
Comment (peterBrxwn) [inline react_native_bridged/current.md:103]: is there a reason why we are deleting Device ID documentation?

File: react_native_bridged/current.md [hunk 2]
```diff
@@ -1058,7 +1048,71 @@ await Countly.initWithConfig(countlyConfig); // Initialize the countly SDK with
...
+<h3 id="h_01HBZPWR8E1BF4J850A5VB9BGJ">Manual Reporting</h3>
+<p>
+  If you have a custom UI where you collect user feedback or you already have some
+  feedback data in your system you can report that information to specific feedback
+  widgets that you have created in your dashboard through the SDK manually.
+</p>
+<p>This processes consists of three steps:</p>
+<ul>
+  <li>
+    Retrieving the list of available widgets and picking one. This is the same
```
Comment (peterBrxwn) [inline react_native_bridged/current.md:1061]: Can we rephrase this sentence (starting from "This is the same initial...)

File: react_native_bridged/current.md [hunk 3]
```diff
@@ -1058,7 +1048,71 @@ await Countly.initWithConfig(countlyConfig); // Initialize the countly SDK with
...
+    initial step while using the feedback widgets and reuses the same call to
+    retrieve them.
+  </li>
+  <li>
+    Download widget data from the server (here you would use a widget you pick
```
Comment (peterBrxwn) [inline react_native_bridged/current.md:1066]: The first item on the list is "Retrieving...". So this should be "Downloading" and the last should be "Reporting"

Comment (turtledreams) [reply react_native_bridged/current.md:1066]: or maybe first one can be "retrive". which one do you like the most?

Comment (peterBrxwn) [reply react_native_bridged/current.md:1066]: Yeah, we can use retrieve

**Pull request merged**